draft-kazuho-httpbis-priority.md

```diff
@@ -98,6 +98,62 @@ The terms sh-token and sh-boolean are imported from
 Example HTTP requests and responses use the HTTP/2-style formatting from
 {{?RFC7540}}.
 
+# Negotiating Priorities
+
+The HTTP/2 specification defines a priority scheme in [RFC7540],
+Section 5.3, which some implementers have opted not to fully support.
+The lack of signalling about the status of the implementation has
+caused several implementations to implement heuristics to detect when
+the clients they are connected to do not support priorities as
+defined and take steps to compensate for that.  The intent of this
+negotiation is to provide an affirmative signalling mechanism for each
+peer to communicate which, if any, priority schemes are supported,
+including the priority scheme as defined in [RFC7540], Section 5.3.
+
+For both HTTP/2 and HTTP/3, either peer's SETTINGS may arrive first,
+so any negotiation must be unilateral and not rely upon receiving
+the peer's SETTINGS value.
+
+Implementations are likely to only use one prioritization scheme at
+once, so the SETTINGS SHOULD be sent prior to the first request.
+In HTTP/3, the SETTINGS may arrive after the first request even if
+they are sent first.  In order to avoid the server choosing
+incorrectly in HTTP/3, the client SHOULD only send priority
+information via its most preferred scheme until it knows what is
+supported by the server.
+
+## The SETTINGS_PRIORITIES SETTINGS Parameter
+
+This document adds a new SETTINGS parameter to those defined by
+[RFC7540], Section 6.5.2.
+
+The new parameter name is SETTINGS_PRIORITIES, which allows both
+peers to indicate which prioritization schemes they support.
+
+A value of 0 indicates no support for priorities. If either side sends the
+parameter with a value of 0, clients SHOULD NOT send priority frames
+and servers SHOULD NOT make any assumptions based on the presence or
+lack thereof of priority frames.
+
+If the value is non-zero, then the least significant 8 bits indicates the
```

Comment:
What if the 8 least significant bits are 0 but the next 8 bits are not?

Reply:
Added below.